draft-yoshino-wish-00.txt

```diff
@@ -321,6 +257,70 @@ Internet-DrafWiSH: A General Purpose Message Framing over B October 2016
    needed.  What contents are exchanged and in what encoding they are
    exchanged over WiSH are to be defined by the application layer.
 
+6.  Using WiSH over HTTP
+
+   Standard HTTP (REST) semantics should be followed, especially the
+   choice of the HTTP method.  Some HTTP semantics may not be
+   applicable, e.g.  Cache-Control, when the body is streamed.  However,
+   such limitation is not specific to WiSH.
+
+7.  WebSocket Compatibility
+
+   In this section, we discuss how to bridge WiSH and WebSocket (API and
```

Comment:
remove "(API and protocol)"

This section lists all the known issues that need be addressed to bridge WiSH (over HTTP) and WebSocket.